code/AEPSDK.brs

```diff
@@ -226,12 +232,96 @@ function AdobeAEPSDKInit() as object
             data[m._private.cons.EVENT_DATA_KEY.ecid] = ecid
             event = _adb_RequestEvent(m._private.cons.PUBLIC_API.SET_EXPERIENCE_CLOUD_ID, data)
             m._private.dispatchEvent(event)
-        end function
+        end function,
+
+        ' ****************************************************************************************************
+        '
+        ' Call this function to start a new Media session with the given XDM data. The XDM data must be the type
+        ' of "media.sessionStart".
+        ' If the "playerName", "channel", and "appVersion" are not provided in the XDM data, the SDK will use
+        ' the global values passed via "updateConfiguration" API.
+        '
+        ' @param xdmData as object                  : the XDM data of type "media.sessionStart"
+        ' @param [optional] configuration as object : the session-level configuration
+        '
+        ' ****************************************************************************************************
+        createMediaSession: function(xdmData as object, configuration = {} as object) as void
+            _adb_logDebug("API: createMediaSession()")
+
+            if m._private.mediaSession.isActive()
+                _adb_logError("createMediaSession() - The previous media session is not ended correctly, try to end it now.")
```

Comment:
nit: Aborting previous media session as it is still active. 
Can you also make this a warning?

Reply:
@addb  suggested updating logs to `Ending previous session before starting a new one.` 
I'm ok with either one. I will Arjun's suggestion for now. @praveek Let me know if you have any objections.